checks/evaluation/pinned_dependencies.go

```diff
@@ -32,7 +32,7 @@ type pinnedResult struct {
 
 // Structure to host information about pinned github
 // or third party dependencies.
-type worklowPinningResult struct {
+type workflowPinningResult struct {
```

Comment:
I believe that lowercase items are deemed private. I've avoided changing one or two items with capital names...